### ISSUE TITLE
Bug Report: highway=road Rendering Issue

### ISSUE THREAD

**Issue by cmoffroad** [osmandapp/OsmAnd#20357]
### Description

I would like to bring to your attention an important issue regarding your rendering of unknown roads (highway=road).

<img width="119" alt="Screen Shot 2567-07-20 at 16 35 43" src="https://github.com/user-attachments/assets/2aa99da4-243d-4bcd-b555-f57a6168d2e1">

According to the OpenStreetMap wiki, the tag highway=road is intended for temporary use for roads/paths that have not yet been surveyed.

This can encompass a wide range of paths, from hiking trails to major motorways.

When a particular renderer needs to display these roads, it is advisable to make them visually distinct from other rendered ways to avoid confusion.

<img width="573" alt="Screen Shot 2567-07-20 at 16 49 30" src="https://github.com/user-attachments/assets/86d70407-3d50-4a62-b928-1b1a57a2220c">

For an outdoor-focused renderer like yours, there is no necessity to display highway=road tags. Displaying these unsurveyed or potentially erroneous roads and trails could mislead end-users.

For reference, Komoot, LocusMap, maps.me and now Strava, have already implemented solutions to hide these tags by default. We recommend adopting a similar approach to enhance the accuracy and reliability of your maps.

Sources:

- https://wiki.openstreetmap.org/wiki/Tag:highway%3Droad
- https://community.openstreetmap.org/t/new-lifecycle-tag-unconfirmed-highway/116102/26

### Steps to reproduce

1. Open the Overpass Query using this link: https://overpass-turbo.eu/s/1OIG
2. Zoom out to your area of interest and click "Run." 
3. Zoom in on each `highway=road` segment from the results. 
4. Copy the zoom level and coordinates, then apply them to the Strava Map URL.
5. The `highway=road` segments appear as minor paved roads. 
6. These segments should not be visible until an OSM mapper surveys and classifies or removes them.

Sample:
- Open this OSM Way for reference: https://www.openstreetmap.org/way/89185730/history
- Navigate to this segment at zoom 16, coordinates: 18.2289, 98.6966

### Actual result

<img width="119" alt="Screen Shot 2567-07-20 at 16 35 43" src="https://github.com/user-attachments/assets/2aa99da4-243d-4bcd-b555-f57a6168d2e1">

Currently, it appears as a large paved road.

### Expected result

Ideally, segments tagged with `highway=road` should not be visible by default. If there is an option to enable them (e.g., OSMMapperAssistant), the rendering should be distinctly different from all other highway classifications, such as a thin gray line.

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version:
Android/iOS version:
Device model:
Crash-Logs: ?
```

**Comment by cmoffroad** [comment]
Any chance you could look into this ? Other outdoor platforms like Strava, Mapy, GuruMaps, and Maps.me have already rolled out a fix. Thanks a bunch in advance! 🙏🙏🙏

**Comment by vshcherb** [comment]
I'm afraid it's out of scope for default rendering style

**Comment by cmoffroad** [comment]
> I'm afraid it's out of scope for default rendering style

Is that a technical limitation, or is there something else?
I'm a bit surprised because all the other apps I reached out to had no issues and agreed to hide them.

**Comment by sonora** [comment]
Fixed for the **Touring view** renderer in https://github.com/osmandapp/OsmAnd-resources/commit/cf52dd690c844070a0ed697af0d279c8f69d342f.

`highway=road` is not hidden by default, but now rendered distinguishable (dash style as in `highway=proposed` and `highway=construction`, but separate gray colors.

**Comment by sonora** [comment]
PS: It aeems the OSM naming "road" for the value (or how the value is nowadays recommended only for "fixme" cases), is really unfortunate. The multitide of uses and also of renderers not compliant with how the OSM wiki nowadays prescribes using it prove that it is rather counterintuitive to try to convey the _alerting_ meaning "unsurveyed" by the _inconspicouos_ expression "road".

**Comment by sonora** [comment]
Inspecting our default renderer (where fixing this would fix it for most other of our map styles, as many dependent renderers do not have separate code for `highway=road`):
- We allow rendering of `highway=road` _as a area_, which should probably be removed, the OSM wiki allows the k/v combination only for _lines_
- We have one (dedicated) code section where `highway=road` is handled (for lines), so a modification there to produce a separate appearance (like with a distinguishable color and/or dash pattern) would not seem complicated. Question is more how distinguishable exactly we would want things.

Check the legend for Touring view to see how I have done it there:
<details>
  <summary>Click to expand</summary>
  <img src="https://github.com/user-attachments/assets/a350aee1-cf52-419f-b9ba-63e950da575c"  width="300" height="auto"/> </td>
</details>

**Comment by cmoffroad** [comment]
> Check the legend for Touring view to see how I have done it there:

Looks great! Do you have any visual samples on how construction/proposed looks like ?

**Comment by sonora** [comment]
`construction` is the same in 'construction orange and white' :wink:, and `proposed` (just notice it seems not part of our automated legend somehow, but I have strawmanned it below) is the same but white replaced by gray:

  <img src="https://github.com/user-attachments/assets/ae063aed-af6a-4d50-9b5b-8d09951a8c5b"  width="300" height="auto"/> </td>

Please note that this is for OsmAnd's 'Touruing view' map style, but OsmAnd comes with a variety of user selectable map styles.

**Comment by cmoffroad** [comment]
> I'm afraid it's out of scope for default rendering style

@vshcherb Could we at least drop the shadowColor from `highway=road`?

I fully support keeping `highway=road` visible in OsmAnd to encourage mappers to survey and reclassify them. 

> So I would say only fixing that one ([code ref](https://github.com/osmandapp/OsmAnd-resources/blob/98cff35177f7349bbaeb1bdbe11b6fa5b5ff4561/rendering_styles/default.render.xml#L10369)) may probably finally justify closing this issue.

However, the current rendering lumps them into the residential/unclassified category, which is misleading. Keep in mind that this tag could apply to anything from a tollway to a footpath.

The border makes it look like the road has a defined function, but that's not true. A simple light gray would do the job. In fact, even rendering additional tags like bridge, tunnel, or covered is unnecessary here—`highway=road` already signals to end-users that the way needs resurveying and re-tagging, as it may include invalid data.

Separating this into its own rendering section would fix the issue. I'm happy to submit a pull request to make that happen.

**Comment by cmoffroad** [comment]
FYI, I’ve submitted a pull request to improve the handling of `highway=road` in `default.render.xml`.

Before: <img width="1280" alt="road-rendering-before" src="https://github.com/user-attachments/assets/b1f51bb0-64a7-4304-b831-5227bf241671" />
After: <img width="1280" alt="road-rendering-after" src="https://github.com/user-attachments/assets/685ac832-e5d8-4d49-bb35-e32d43a030b4" />

**Comment by Zirochkabila** [comment]
# Testing 1
`18.672478, 98.578514`

`OsmAnd~ 5.0.0#4079m, Released: 2025-01-08` & `WEB`- **Add**
`iOS4.9.3.4936` - Waiting for

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/8f874acb-e4e4-4ea8-8ebb-11253ef7a721"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/37c7ad73-214d-4edf-8141-ff2108fe3440"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/87c108e5-14ff-4610-86e5-dca547124e70"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by dmpr0** [comment]
# Review

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/c70cf277-5733-4807-8961-eca488a3b064"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/user-attachments/assets/325f7002-ffbf-499b-9670-e1b5b667a77c"  width="300" height="auto"/> </td>
  </tr>
</table>  


- [ ] Hardly visible in OsmAnd Map style with Night mode, especially over forest/wood. https://www.openstreetmap.org/way/1031227729

**Comment by cmoffroad** [comment]
> # Review
> <img alt="" width="300" height="auto" src="https://private-user-images.githubusercontent.com/9365308/409578027-c70cf277-5733-4807-8961-eca488a3b064.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg2NzkxMDcsIm5iZiI6MTczODY3ODgwNywicGF0aCI6Ii85MzY1MzA4LzQwOTU3ODAyNy1jNzBjZjI3Ny01NzMzLTQ4MDctODk2MS1lY2E0ODhhM2IwNjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDIwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAyMDRUMTQyMDA3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjA4NTk0YmNmNmQ2MDk2NTY0MTdkZGI4YmVlNWQxZGIxNTI3M2M5NGNlNTZmYThhYWEwZmY0ZjNkYzRhZmIyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.0__0bgM7MIyeVVxOrC-w39v5GfpxdlbPJOAxuOXPOfU">	<img alt="" width="300" height="auto" src="https://private-user-images.githubusercontent.com/9365308/409578028-325f7002-ffbf-499b-9670-e1b5b667a77c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg2NzkxMDcsIm5iZiI6MTczODY3ODgwNywicGF0aCI6Ii85MzY1MzA4LzQwOTU3ODAyOC0zMjVmNzAwMi1mZmJmLTQ5OWItOTY3MC1lMWI1YjY2N2E3N2MucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDIwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAyMDRUMTQyMDA3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTA4NTRiOGM1YjIyMmQ5MjNjMDBkYmJlNGNmN2QyZTZjMzBjZGZhMjlmOTE0ZjU1MDJiYzRkZDAzZmEzNTY4ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.KPNMBejet3uUjocaOtiucdDZoGEbOSJY3cfRy48wvA8">
> * [ ]  Hardly visible in OsmAnd Map style with Night mode, especially over forest/wood. https://www.openstreetmap.org/way/1031227729

I have submitted a PR with color `#3a3a3a`, but if you need something brighter, you can try `#404040` or `#4a4a4a`.

It's important that these roads don't draw too much visual attention. Unlike others, they require careful consideration due to the potential for erroneous data.

https://github.com/osmandapp/OsmAnd-resources/pull/1181

**Comment by xmd5a2** [comment]
I think this is enough

![Image](https://github.com/user-attachments/assets/cf729da1-5866-4b1b-b5a1-c944b469b10c)

**Comment by Zirochkabila** [comment]
# Testing 2
`4.9.5.4976`
`OsmAnd~ 5.0.0#4393m, Released: 2025-02-17`
Fixed
